src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -511,6 +504,15 @@ export default defineComponent({
         })
 
         this.player.one('loadedmetadata', () => {
+          if (this.useDash) {
+            // this.dataSetup.plugins.httpSourceSelector = {
+            // default: 'auto'
+            // }
+
+            // this.player.httpSourceSelector()
```

Comment:
???
Add code comment to explain or remove?

I know it's just moved from old place but if they are not used since commented maybe it should be removed entirely?

Reply:
Preston explained not to long ago on in the Matrix chat that the code is there, if we want to switch back to using videojs-http-quality-selector as the quality selector instead of having a custom DASH quality selector inside FreeTube (yes we currently still import the dependency but we don't use it).

I can get rid of it if you really want me to, although I have to strip out a lot more commented out code that is related to it.

Reply:
I am ok with keeping commented code but some code comments should be added
Something like `reserved for switching back to videojs-http-quality-selector if needed`
I think it's healthier to not have to ask every time I see commented code